src/app/admin/components/add-evaluator/add-evaluator.component.scss

```diff
@@ -1,4 +1,4 @@
-@import '~_vars.scss';
+@import 'vars.scss';
```

Comment:
Why did this change? Just curious.

Reply:
I didn't look into angular docs but the compiler threw an error on the '~_' format. I just removed that and everything compiled correctly.